README.md

```diff
@@ -47,3 +47,23 @@ Decode a binary:
             97,42,106,106>>,
     bert:decode(Bin).
     > #{key => [{some, "tuple"}, {true, 42}]}
+
```

Comment:
It would be great to mention about running property tests with `rebar3 as test proper`. What do you think?

Reply:
Yes. I add about it in README in Test subsection.